pages/_app/index.jsx

```diff
@@ -79,8 +80,10 @@ class App extends NextApp {
     }
   }
 
-  redirectToLogin = path => {
-    window.location.replace(getLoginPage(path))
+  redirectToLogin = pathname => {
+    const url = new URL('/')
```

Comment:
This will fail. You need to pass full URL or pass base as a second param.

```
new URL('/')
VM131:1 Uncaught TypeError: Failed to construct 'URL': Invalid URL
    at <anonymous>:1:1
```

Reply:
I guess it's server side rendering problem, is not it?